docs/docs/messaging-and-voice-channels.mdx

```diff
@@ -59,3 +59,13 @@ With the free-tier of ngrok, you can run into limits on how many connections you
 As of writing this, it is set to 40 connections / minute.
 
 :::
+
+Alternatively you can make your assistent listen on a specific address using the `-i` command line 
```

Comment:
Can we also add something to `docs/docs/command-line-interface.mdx`?

Reply:
Did that!